design/ceph/object-store-realm.md

```diff
@@ -0,0 +1,169 @@
+# Rook Object Store Realm
+
+## Prerequisites
+
+A rook ceph cluster without object-stores.
+
+## Ceph Multisite Overview
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+### Ceph Multisite data model
+
+For reference, here is a description of the underlying Ceph Multisite data model.
+
+```
+A cluster has one or more realms
+
+A realm spans one or more clusters
+A realm has one or more zonegroups
+A realm has one master zonegroup
+A realm defined in another cluster is replicated with the pull command
+The objects in a realm are independent and isolated from objects in other realms
```

Comment:
what about buckets, zones and zonegroups? are they also isolated between realms?